tests/fixtures/js-stubs/types.tsx

```diff
@@ -0,0 +1,142 @@
+type SimpleStub<T = any> = () => T;
+
+type OverridableStub<T = any> = (params?: Partial<T>) => T;
+
+type OverridableStubList<T = any> = (params?: T[]) => T[];
+
+type TestStubFixtures = {
+  AccessRequest: OverridableStub;
+  AccountAppearance: OverridableStub;
+  AccountEmails: OverridableStubList;
+  ActivityFeed: OverridableStub;
+  ApiApplication: OverridableStub;
+  ApiKey: OverridableStub;
```

Comment:
Should these generics be filled out as the implementations are all JS only files.

Reply:
I would love for them to be. But it's going to be a lot more effort to make types align in our tests :( we probably should do them one at a time